src/mapeo-project.js

```diff
@@ -86,7 +99,31 @@ export class MapeoProject {
       config: new DataStore({
         coreManager: this.#coreManager,
         namespace: 'config',
-        batch: (entries) => indexWriter.batch(entries),
+        batch: async (entries) => {
```

Comment:
```suggestion
        batch: (entries) => this.#handleConfigEntries(entries)
```
Just in the name of keeping the constructor method easier to read, since it's getting quite long.

Reply:
addressed via https://github.com/digidem/mapeo-core-next/pull/187/commits/bd311a2de61c3fc1d42a03e34ad1f3538846f232